src/main.cpp

```diff
@@ -9,13 +9,30 @@
 #include "widget/canvas.hpp"
 
 #include "widget/toolSelector.hpp"
+#include "file/serializer.hpp"
 
 int main (int argc, char **argv){
 	QApplication a(argc, argv);
 	QGraphicsScene scene;
 	Lipuma::Canvas *canvas = new Lipuma::Canvas(&scene);
 	QMainWindow *mainWin = new QMainWindow();
 	mainWin->setCentralWidget(canvas);
+	QMenu* fileMenu = mainWin->menuBar()->addMenu("File");
+
+	QAction* saveAction = fileMenu->addAction("&Save As");
```

Comment:
Is the prepended ampersand intended here?

Reply:
That is a feature of actions that specify what the associated hotkeys should be.